app/controllers/admin/neurons_controller.rb

```diff
@@ -8,7 +8,7 @@ class NeuronsController < Neurons::BaseController
       if current_user.admin?
         Neuron.all
       else
-        Neuron.active
+        Neuron.where.not(state: deleted_state)
```

Comment:
_note for self_

See how can this be moved up to cancan so that abilities handle scopes rather than controllers